.github/workflows/playwright.yml

```
@@ -66,6 +66,7 @@ jobs:
           NODE_OPTIONS: "--max-old-space-size=8192"
           VITE_COVERAGE: "true"
           COVERAGE: "true"
+          NODE_OPTIONS: "--max-old-space-size=8192"
```

actionlint (1.7.4)

key "NODE_OPTIONS" is duplicated in env. previously defined at line:66,col:11. note that this key is case insensitive

(syntax-check)